tests/test_core.py

```diff
@@ -32,11 +32,20 @@ def test_Aggregate():
     data_dir = "tests/resources/datasets/marco"
     A = C.datasets["marco"].get_audio_numpy("impulse_response+90d", "OCT3D")
     wavs_dir = os.path.join(data_dir, "3D-MARCo Impulse Responses/01_Speaker_+90deg_3m")
-    wavs = os.listdir(wavs_dir)
+
+    wavs = [
+        wavs_dir + "/+90deg_010_OCT3D_1_FL.wav",
+        wavs_dir + "/+90deg_011_OCT3D_2_FR.wav",
+        wavs_dir + "/+90deg_012_OCT3D_3_FC.wav",
+        wavs_dir + "/+90deg_013_OCT3D_4_RL.wav",
+        wavs_dir + "/+90deg_014_OCT3D_5_RR.wav",
+        wavs_dir + "/+90deg_015_OCT3D_6_FLh_1m.wav",
+        wavs_dir + "/+90deg_016_OCT3D_7_FRh_1m.wav",
+        wavs_dir + "/+90deg_017_OCT3D_8_RLh_1m.wav",
+        wavs_dir + "/+90deg_018_OCT3D_9_RRh_1m.wav",
+    ]
```

Comment:
can you please write this using list comprehension? i.e. first create a list of filenames (example `wavs = ['+90deg_010_OCT3D_1_FL.wav', '+90deg_011_OCT3D_2_FR.wav', ...]` and then use list comprehension to join `wavs_dir` with the filenames directly in the next line

`B = np.array([librosa.load(w, sr=48000, mono=False)[0] for w in wavs])`

you can use `os.path.join` to concatenate the path and each filename inside the list comprehension.